eox_tagging/admin.py

```diff
@@ -71,7 +71,7 @@ def tagged_object(self, tag):
         except Exception as error:
             return str(error)
 
-    def add_view(self, request, *args, **kwargs):  # pylint: disable=arguments-differ
+    def add_view(self, request, *args, **kwargs):  # pylint: disable=signature-differs
```

Comment:
do you know why we are not using the same signature ?

Reply:
I'm not really sure about this one so I'm going to revert to the original signature:
`ModelAdmin.add_view(request, form_url='', extra_context=None)`